src/coreclr/src/vm/ecalllist.h

```diff
@@ -132,13 +132,9 @@ FCFuncStart(gDiagnosticsStackTrace)
 FCFuncEnd()
 
 FCFuncStart(gDateTimeFuncs)
-#if !defined(TARGET_UNIX)
-    FCFuncElement("GetSystemTimeWithLeapSecondsHandling", SystemNative::GetSystemTimeWithLeapSecondsHandling)
-    FCFuncElement("ValidateSystemTime", SystemNative::ValidateSystemTime)
-    FCFuncElement("FileTimeToSystemTime", SystemNative::FileTimeToSystemTime)
-    FCFuncElement("SystemTimeToFileTime", SystemNative::SystemTimeToFileTime)
-#endif // TARGET_UNIX
+#if defined(TARGET_UNIX)
```

Comment:
It should be trivial to enable the managed implementation on Unix too.

Reply:
@jkotas The Unix implementation is [here](https://github.com/dotnet/runtime/blob/master/src/libraries/Native/Unix/System.Native/pal_datetime.c), but I don't know how to translate this to C#/CoreLib, especially with that ifdef. Do you have a suggestion for who on the team I can lean on for expertise here?

Reply:
- Delete this ifdef https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/DateTime.Unix.cs#L10 to enable the managed impl on all runtimes
- Delete this FCall
- Maybe mark `SystemNative_GetSystemTimeAsTicks` with SuppressGCTransition